src/components/Tool/Appeal/AppealsContext/AppealsContext.tsx

```diff
@@ -16,6 +16,7 @@ import { useAccountListId } from 'src/hooks/useAccountListId';
 import { useMassSelection } from 'src/hooks/useMassSelection';
 import { sanitizeFilters } from 'src/lib/sanitizeFilters';
 import { useContactsQuery } from './contacts.generated';
+import { useContactsCountQuery } from './contactsCount.generated';
```

Comment:
I think I've reviewed some of this code before?

Reply:
Yes, some of this will be in other PRs as it's a core piece of functionality and has been edited multiple times